.golangci.yaml

```diff
@@ -1,5 +1,5 @@
 issues:
-  new: true
+  new-from-rev: main
```

Comment:
This makes local linting results similar to the pipeline linting results

Reply:
works for me 👍